lib/utils.ts

```diff
@@ -47,6 +47,19 @@ export function generateBuildTasks(
 			if (img.image.context == null) {
 				throw new Error('Must have a context specified with a Dockerfile');
 			}
+			// It's possible to specify an image name as well as
+			// a build, but that doesn't make sense in a balena
+			// ecosystem, so we remove it.
+			// We also remove anything that goes into dockerOpts
+			const {
+				image,
```

Comment:
Build is failing on this line:
```
lib/utils.ts:55:5 - error TS2339: Property 'image' does not exist on type 'BuildConfig'.

55     image,
       ~~~~~
```
Because, I think, `img.image.image` is not a string here, it's a `BuildConfig` object that indeed does not have a property named `image`:
```
export interface BuildConfig {
    args?: Dict<string>;
    cache_from?: ListOfUniqueItems<string>;
    context: string;
    dockerfile?: string;
    extra_hosts?: ListOfUniqueItems<string>;
    isolation?: string;
    labels?: Dict<string>;
    network?: string;
    shm_size?: number | string;
    tag?: string;
    target?: string;
}
```